.github/ISSUE_TEMPLATE/task.yml

```diff
@@ -0,0 +1,50 @@
+name: Task
+description: Describe a generic activity we should carry out.
+labels: ["kind/task"]
+body:
+  - type: dropdown
+    id: Component
+    attributes:
+      label: Component
+      description: In which component did the issue occur?
+      options:
+        - Not sure*
+        - Operate*
+        - Optimize*
+        - Tasklist*
+        - Identity*
+        - Zeebe*
+        - C8Run*
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: what
+    attributes:
+      label: What should we do?
+      description: Clearly describe the activity we should carry out.
+    validations:
+      required: true
+  - type: textarea
+    id: why
+    attributes:
+      label: Why should we do it?
+      description: Argue why doing it is a healthy investment of our time.
+    validations:
+      required: true
+  - type: textarea
+    id: acceptance
+    attributes:
+      label: Acceptance criteria
+      description: add the TODOs in here (optional).
+      value: |
+        - [ ] ...
+    validations:
+      required: false
+  - type: textarea
+    id: links
+    attributes:
+      label: Support ticket link
+      description: Add link to the support ticket here if applicable.
+    validations:
+      required: false
```

Comment:
Please add a section "Breakdown" here (not required, not prefilled).

Reply:
@ThorbenLindhauer would you say this is kinda "scratch pad" for the breakdown, as we probably use the "sub issues" feature for the actual breakdown, right?

Reply:
Sub issues should be sufficient, so we can remove this. Please also see my comment above.